common/pb/response_ban_history.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto2";
+import "response.proto";
+
+message Response_BanHistory{
+    extend Response {
+        optional Response_BanHistory ext = 1012;
+    }
+    repeated string admin_id = 1;
+    repeated string admin_name = 2;
+    repeated string ban_time = 3;
+    repeated string ban_length = 4;
+    repeated string ban_reason = 5;
+    repeated string visible_reason = 6;
```

Comment:
Thinking through this, is this the behavior you want?  I dont think it is.  The server inst keeping any type of constant track of ban history.  The server info pb items are all based around objects that have specific sets of information that are stored and manipulated in memory requiring the information to be retained for periods of time during specific execution points when dealing with multi user bits of information.  So for example the serverinfo_card is keeping track of card information state during a game.  The ban history of a user doesn't really fall into that category.  We are looking for the data for a response from a command. The server won't be keeping any state change of the ban history nor will it be passed to anyone but the person making the command request.